modules/local/alevinqc.nf

```diff
@@ -1,23 +1,25 @@
 process ALEVINQC {
```

Comment:
Is there some local modules that can be removed now that you switched to nf-core modules?